tests/unit/CIGitLogicTest.ts

```diff
@@ -20,6 +20,7 @@ import * as Log from '../../scripts/utils/Logger';
 
 const DUMMY_DIR = path.resolve(os.homedir(), 'DumDumRepo');
 const GIT_REMOTE = path.resolve(os.homedir(), 'dummyGitRemotes/DumDumRepo');
+const SUBMOD_REMOTE = path.resolve(os.homedir(), 'dummyGitRemotes/SubMod');
```

Comment:
Might the logic that simulates git submodule commands be closer to reality if it was instead `'DumDumRepo/SubMod'? Just a thought, perhaps being close to 'real' workflow commands isn't important.